### PR TITLE
fix(ctb): MIPS Styling and Docs

### PR DESCRIPTION
**Pull request by refcell**
**Description**

Stylizes MIPS.sol and adds some documentation in the form of inline comments.

### CODE REVIEW

**Comment by refcell** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #6269** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6269" target="_blank">Graphite</a> 
    * **PR #6270** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6270" target="_blank">Graphite</a> 
      * **PR #6276** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6276" target="_blank">Graphite</a>  👈
        * **PR #6293** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6293" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6276?utm_source=stack-comment).

File: packages/contracts-bedrock/contracts/cannon/MIPS.sol [hunk 1]
```diff
@@ -3,625 +3,879 @@ pragma solidity 0.7.6;
...
+          state.hi = uint32(int32(_rs) % int32(_rt));
+          state.lo = uint32(int32(_rs) / int32(_rt));
+        }
+        // divu: Unsigned divides `rs` by `rt`.
+        // Stores the quotient in LO
+        // And the remainder in HI
+        else if (_func == 0x1b) {
+          state.hi = _rs % _rt;
+          state.lo = _rs / _rt;
+        }
```
Comment (clabby) [inline packages/contracts-bedrock/contracts/cannon/MIPS.sol]: Tabs look to have been messed up here

**Review by clabby — CHANGES_REQUESTED** [review]
LGTM; There is an odd mix of 2 spaces / 4 spaces indentation. LGTM after that's fixed :)

**Review by clabby — APPROVED** [review]
👑 

g2g once CI passes.

**Pull request merged**